odoo/addons/base/models/assetsbundle.py

```diff
@@ -377,47 +383,57 @@ def is_css_preprocessed(self):
     def preprocess_css(self, debug=False, old_attachments=None):
         """
             Checks if the bundle contains any sass/less content, then compiles it to css.
+            If user language direction is Right to Left then also consider css fiels and call run_rtlcss,
+            If debug=assets then we will also store css files in ir.attachment with rtl suffix so later it is served from attachment
```

Comment:
@msh-odoo could you elaborate on this part? you mean the debug=assets will no longer server original files?

Reply:
@mart-e What current implementation does, it store all files in ir.attachment which needs to be processed by preprocessor like scss, less so that later we don't need to process it and we fetch from ir.attachment, note that rtlcss is also a preprocessor and when we are in debug=assets mode, each file is separately stored while css files are not going to processed as it does not need preprocessing but in case of Right to left we need process css files(run rtlcss for css files also)